src/registrar/assets/src/sass/_theme/_typography.scss

```diff
@@ -34,6 +34,9 @@ h2 {
 .usa-form,
 .usa-form fieldset {
   font-size: 1rem;
+  .usa-legend {
```

Comment:
Now that our field headings and subheadings are moved to the legends in the screenreader fix, we're updating the font size of our legends to maintain our font size of 16px (originally 16.96px from usa-legend font size default being 1.06rem)